docs/Home.md

```diff
@@ -119,6 +119,19 @@ The following permissions will grant you enough permissions to deploy to your cl
     ```bash
     kubectl create -f PATH_TO_YOUR_ROLE_BINDING_YAML
     ```
+##### Automation Option
+For the more restrictive permissions option, we offer an example automated solution for configuring RBAC permissions.
```

Comment:
I would add some prose to this description detailing the overall strategy so it's clear to customers what the automation is doing. Specifically I'd suggest mentioning the robot deployer role which is created in the cluster, as well as the IAM role which is created.

Reply:
Done

Reply:
Suggest a slight modification: "The overall strategy is to grant your GCP service account an IAM role with read-only access to clusters within GKE, as well as a RBAC cluster role binding with limited access to deploy resources in Kubernetes."

Reply:
Done.